localMatchApp/test/tst_logintest.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <gmock/gmock-matchers.h>
+#include <gtest/gtest.h>
+#include <iostream>
+#include <string>
+#include "Login.h"
+using namespace testing;
+
+TEST ( LoginTest, TestPassword ) {
+  Login prueba;
```

Comment:
Why does the test logic is inside a .h file?

Reply:
Because there is a .cpp file that includes the test and runs it, so everyone can create a test file .h and then add it in the .cpp file.